setup.py

```diff
@@ -15,38 +16,40 @@
 import platform
 import sys
 
-platform_system = platform.system() # Linux: Linux; Mac:Darwin; Windows: Windows
-
 here = path.abspath(path.dirname(__file__))
-blast_path_bin = path.join(here, 'gff3tool', 'lib', 'ncbi-blast+', 'bin')
-blast_path = path.join(here, 'gff3tool', 'lib', 'ncbi-blast+')
-blast_file = path.join(blast_path, 'blast.tgz')
-
-if not path.exists(blast_path):
```

Comment:
It seems to me that it would be better to remain the  path.exists check.

Reply:
I don’t think so. No user will manually create the folder when following the installation step we provide. The issue you met was because git will not delete the folder when checkout in that case.

Reply:
How can you be so sure? If old user pull the new master, won't this issue happen?

Reply:
That’s developer, not user.

Reply:
And it’s a rare event. I don’t think we need to handle such rare event.

Reply:
In the original master, we handle this issue. Is there any reason why we can't remain this feature?

Reply:
As I described in this internal issue, the reason I implemented that it's because original implementation will execute the process of installation twice. Therefore, we need to check the directory exist or not to make the whole process work.